src/App.tsx

```diff
@@ -7,32 +7,17 @@ import { LoginContainer } from './containers/login';
 import { RegistrationContainer } from './containers/registration';
 import { ProjectOverview } from './containers/project-overview';
 
-async function getProjects(supabase: SupabaseClient<Database>) {
-  const { data, error } = await supabase.from('project').select('*');
-
-  if (error) {
-    console.error('Found an issue fetching projects.', error);
-    return Promise.resolve([]);
-  }
-
-  return data;
-}
-
 const App: React.FC = () => {
-  const supabase = createClient<Database>(supabaseUrl, supabaseKey);
-
-  const projects = getProjects(supabase);
-
-  projects.then(console.log);
+  const supabase: SupabaseClient = createClient<Database>(supabaseUrl, supabaseKey);
 
   return (
     <>
       <HashRouter basename = "/">
         <Routes>
-          <Route path="/" element={<LoginContainer supabase={supabase} />} />
+          <Route path="/" element={<ProjectOverview supabase={supabase} />} />
+          <Route path="/overview" element={<ProjectOverview supabase={supabase}/>} />
```

Comment:
We already _have_ a bloody overview page, so I've repurposed it and it looks _lovely_.